scripts/Run-Tests.ps1

```diff
@@ -34,27 +34,15 @@ Import-Module -Force $PSScriptRoot\Projects.psm1
 
 $ScriptName = $([io.Path]::GetFileNameWithoutExtension($PSCommandPath))
 
-$ReporterOption = $null
-if ($AppVeyor) {
-    $ReporterOption = "-appveyor"
```

Comment:
This is dropped without replacement because xunit automatically detects this nowadays (see xunit help parameters). I verified that appveyor was still picking up tests as they ran without this.